src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/IncrementalBuildTest.cs

```diff
@@ -320,14 +320,20 @@ public Class2 ()
 		[Test]
 		public void AppProjectTargetsDoNotBreak ()
 		{
-			var targets = new [] {
+			var targets = new List<string> {
 				"_CopyIntermediateAssemblies",
 				"_GeneratePackageManagerJava",
 				"_ResolveLibraryProjectImports",
 				"_BuildAdditionalResourcesCache",
 				"_CleanIntermediateIfNuGetsChange",
 				"_CopyConfigFiles",
+				"_CopyPdbFiles",
+				"_CopyMdbFiles",
 			};
+			if (IsWindows) {
```

Comment:
This check and the corresponding comment are not correct.  `pdb2mdb` will *not* "only happen[] on Windows."

Prime example: NuGet package contains "legacy"-style `.pdb` file, and NuGet package is used on macOS.  Boom, you need pdb2mdb on macOS.

That might not matter *here*, assuming that the project contains no NuGet references and only uses source-code projects, *and* that `$(DebugType)`=Full (?) on Windows instead of `portable`, so this code *may* still be correct...

Reply:
I explicitly set `DebugType` and made the comment more clear.